portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/snippets/roleSwitch-snippet.vm

```diff
@@ -8,7 +8,6 @@
     #else
         <a id="roleSwitchDropDownToggle" href="javascript:void('0')" aria-hidden='true' aria-haspopup="true" title='${rloader.rs_menuTooltip}'>
             <i class="fa fa-user-secret" aria-hidden="true"></i>
-            <span class='skip'>${rloader.rs_toggleRoleSwitcher}</span>
```

Comment:
Yeah, I'd be fine with adding the property if that improves mobile. I was just wondering if @payten already had a value for the property.